pkg/apis/core/validation/shoot.go

```diff
@@ -1702,6 +1707,15 @@ func ValidateContainerRuntimes(containerRuntime []core.ContainerRuntime, fldPath
 	return allErrs
 }
 
+func ValidateArchitecture(arch *string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if arch != nil && *arch != imagevector.X86 && *arch != imagevector.Arm64 {
```

Comment:
Can we define new constant instead of reusing the ones from the imagevector pkg?